lib/widgets/content.dart

```diff
@@ -812,6 +812,41 @@ class RealmContentNetworkImage extends StatelessWidget {
   }
 }
 
+/// A rounded square with size [size] showing a user's avatar.
+class Avatar extends StatelessWidget {
+  const Avatar({
+    super.key,
+    required this.userId,
+    required this.size,
+    this.borderRadius = 4, // TODO vary default value with [size]?
```

Comment:
I think probably actually cleanest to make this argument required too. That way the value chosen for it is always evident in the source right next to the chosen size.